cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/AbstractSchedulerService.java

```diff
@@ -92,23 +92,24 @@ protected final void startScheduler() {
   protected final void stopScheduler() {
     try {
       streamSizeScheduler.stop();
-      LOG.info("Stopped stram size scheduler");
+      LOG.info("Stopped stream size scheduler");
     } catch (Throwable t) {
-      LOG.error("Error stopping stream size scheduler {}", t.getCause(), t);
+      LOG.error("Error stopping stream size scheduler", t);
       throw Throwables.propagate(t);
     } finally {
       try {
         timeScheduler.stop();
         LOG.info("Stopped time scheduler");
-      } catch (SchedulerException e) {
-        LOG.error("Error stopping time scheduler {}", e.getCause(), e);
+      } catch (org.quartz.SchedulerException e) {
+        LOG.error("Error stopping time scheduler", e);
         throw Throwables.propagate(e);
       }
     }
   }
 
   @Override
-  public void schedule(Id.Program programId, SchedulableProgramType programType, Schedule schedule) {
+  public void schedule(Id.Program programId, SchedulableProgramType programType, Schedule schedule)
+    throws SchedulerException {
```

Comment:
Oh we have a SchedulerException now. My bad, it is because of the order in which I read these changes.